README.md

```diff
@@ -29,9 +31,9 @@ precision computations (for accuracy requirements that exceeds double precision
 This package can be installed using
 
 ```julia
-using Pkg
-Pkg.add("https://github.com/mikelehu/IRKGaussLegendre.jl")
-using IRKGaussLegendre
+julia>using Pkg
+julia>Pkg.add("IRKGaussLegendre.jl")
+julia>using IRKGaussLegendre
```

Comment:
usually it's best to keep it simple and copy-pastable. I'd just remove the julia>